pkgs/applications/networking/browsers/palemoon/mozconfig

```diff
@@ -20,8 +20,6 @@ ac_add_options --enable-strip
 ac_add_options --enable-devtools
 ac_add_options --enable-av1
 
-ac_add_options --disable-eme
-ac_add_options --disable-webrtc
```

Comment:
Only difference in required build flags that I could spot, the application to build wasn't renamed like in v30.